src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -627,6 +635,10 @@ def _db_dw_create(
     Handles common concerns such as setting location and sku properties.
     '''
 
+    if kwargs['compute_model'] == "Serverless":
+        if sku == None or sku.tier == None or sku.family == None or sku.capacity == None:
+            raise CLIError('When creating a severless database, please pass in edition, family, and capacity parameters through -e -f -c')
```

Comment:
This check should not be needed. Tier, family, and capacity can all be specified in the sku name, in which case they don't need to be explicitly provided. This is why we have all the find_capability logic.